opts/options.go

```diff
@@ -5,14 +5,20 @@ import (
 
 	ds "github.com/ipfs/go-datastore"
 	dssync "github.com/ipfs/go-datastore/sync"
+	"github.com/libp2p/go-libp2p-protocol"
 	record "github.com/libp2p/go-libp2p-record"
 )
 
+var ProtocolDHT protocol.ID = "/ipfs/kad/1.0.0"
+var ProtocolDHTOld protocol.ID = "/ipfs/dht"
```

Comment:
I'd leave these variables where they were (they're not really options).

Reply:
Moving those variables back to the `dht` package will create a circular dependency, as the `Defaults` function in the `dhtopts` package needs them.

Reply:
I'm just worried that this is *technically* a breaking change but I guess there isn't really a good way to fix this. Meh, @Kubuxu?

Reply:
Idea: We could move the code in `options.go` into the `dht` package and delete the `dhtopts` package.

Reply:
I put this in a separate package to group the options. Otherwise, they just look like functions and we'd need to call them all `OptionX`, `OptionY` to make it clear that they're *just* options. I'm fine just leaving this as it is. It looks like the primary reason we were exporting those anyways was to allow users to override the protocol names (and this patch provides a better way to do that).

Reply:
> I'm fine just leaving this as it is. 

Sounds good!